rebasehelper/output_tools/html_output_tool.py

```diff
@@ -0,0 +1,369 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os
+import six
+
+from rebasehelper.utils import ProcessHelper
+from rebasehelper.output_tool import BaseOutputTool
+from rebasehelper.results_store import results_store
+from rebasehelper.logger import logger
+
+from yattag import Doc
+#doc, tag, text = Doc().tagtext()
+
+
+class HTMLOutputTool(BaseOutputTool):
+    """ HTML output tool """
+    NAME = 'html'
+
+    @classmethod
+    def match(cls, cmd=None):
+        """
+        Checks if the given string matches the output tool
+
+        :param cmd: output tool name
+        :return: True if the name matches
+        """
+        if cmd == cls.NAME:
+            return True
+        else:
+            return False
+
+    @classmethod
+    def get_name(cls):
+        """
+        Get name of the output_tool
+
+        :return: output tool name
+        """
+        return cls.NAME
+
+    @classmethod
+    def print_report_file_path(cls):
+        """Print path to the report file into the CLI"""
+        cls.colored_output('yellow', '%s report' % cls.NAME)
+        print('%s' % os.path.join(cls.html_path, 'report.' + cls.NAME))
+
+    @classmethod
+    def page_setup(cls):
+        """Generates html menu and loads css styles"""
+        global doc, tag, text
```

Comment:
Again, why global? If you need the variables in other methods why don't you make them class variables?